src/nc/gui/MainWindow.cpp

```diff
@@ -88,7 +88,8 @@ MainWindow::MainWindow(Branding branding, QWidget *parent):
 
     logToken_ = LogToken(logger);
 
-    settings_ = new QSettings(branding_.organizationName(), branding_.applicationName(), this);
+    settings_ = new QSettings(QString(branding.applicationName() + ".ini"), QSettings::IniFormat, this);
```

Comment:
I guess, this will put settings into the current directory, which generally is not going to be writable by the user.